mintapi/api.py

```diff
@@ -59,27 +64,27 @@ def reverse_credit_amount(row):
     return amount if row['isDebit'] else -amount
 
 
-def get_email_code(imap_account, imap_password, imap_server, imap_folder, debug=0, delete=1):
```

Comment:
Won't changing this signature break usage for existing folks?

Reply:
You're right, this change will only update the defaults and test locations. However, the new tests will continue to work for existing int call sites, based on the [python rules for truthiness](https://docs.python.org/2/library/stdtypes.html#truth-value-testing).

For defining the param and showing users how to use it, Boolean is the correct primitive (either it's true/false), not an int.

Reply:
Thanks! What I mean is, if someone is currently calling `get_email_code` with `debug` specified and they upgrade mintapi, their code will create an exception, won't it? I'd prefer for upgrades to not break existing code except in pretty important cases.

I like the idea of making them False/True instead of 0/1, but would prefer that `debug` is kept in its position and a deprecation warning issued for a couple releases instead. Let me know if that makes sense or if I'm misunderstanding. Looks ready to merge after we figure that part out!

Reply:
Sorry; I misunderstood! You're absolutely right. I've added a deprecation warning with instructions on the new method.